src/app/config/features/intranet.config.ts

```diff
@@ -23,6 +24,12 @@ const intranetSchema: Schema<IIntranet> = {
     default: '',
     env: 'INTRANET_IP_LIST_PATH',
   },
+  ogpIpList: {
+    doc: 'Path to file containing list of OGP IP addresses, separated by newlines',
+    format: validateNonIacStringParam,
+    default: '',
+    env: 'OGP_IP_LIST',
```

Comment:
nit: for consistency, should this env be named `OGP_IP_LIST_PATH`?

this might help avoid users being confused and inputting a comma-separated list of IP addresses

Reply:
checked and this is no longer required:
- we're using environment variables directly
- and moving away from EFS-stored configuration files (Elastic Beanstalk legacy setup)